README.md

```
@@ -43,6 +43,21 @@ node            |  |                               |  |
+### Docker
+
+1. Install [Docker](https://www.docker.com/get-started) 20.10 or higher
+
+2. Pull the latest celestia-app Docker image
+    ```bash
+    docker pull ghcr.io/celestiaorg/celestia-app:latest
+    ```
+
+3. Run the application
+- Navigate to `docker` directory, run:
+    ```bash
```

Markdownlint (0.37.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)